distributed/tests/test_system.py

```diff
@@ -87,13 +88,15 @@ def myopen(path, *args, **kwargs):
     assert limit == 10
 
 
+@pytest.mark.xfail(
+    MACOS,
+    reason="MacOSX github CI raises 'ValueError: current limit exceeds maximum limit' "
```

Comment:
```suggestion
    reason="Mac OS raises 'ValueError: current limit exceeds maximum limit' "
```

I can reproduce this failure locally, my guess is that we can be more general with this statement.